joomla/joomla-cms
# [com_fields] Add option to show or hide field label

Pull Request for Issue #13707 .

### Summary of Changes
I added an option in the params area to hide the field label. The switch is worded so it's a Hide switch not a Show switch, and the default is No so the label appears by default.

### Testing Instructions
Install the patch, create two labels, or pretty much any kind.
Enter some data in the labels in an article so they appear on the front end of the site
Set one field to Hide the label, leave the other as the default
You should see one field showing the label, and the other not showing it. 
### Documentation Changes Required
not sure yet
